extras/http2/src/stylesheets/components/README.md

```diff
@@ -0,0 +1,27 @@
+# HTTP2 component styles
+
```

Comment:
This readme focuses on "this is how you _must_ organize your files", but doesn't say how that all relates to the task that linked to this from the main README. What about spinning the same information in a less-"should" way, positioning as "if you're using http2, you can sprinkle stylesheet links through your page without screwing your performance, and doing that will benefit you because then each page can have just the css it needs, so we provide a task that provides oob support for that model. here's an example of how you might set it up:"